plugins/modules/system/xfconf.py

```diff
@@ -13,6 +13,7 @@
 author:
     - "Joseph Benden (@jbenden)"
     - "Alexei Znamensky (@russoz)"
+    - "Adriaan Callaerts (@call-a3)"
```

Comment:
@call-a3 First of all, thank you very much for your contribution! :-)
I am not entirely sure of the semantics here at the "author" - being new to it myself - but I believe this is supposed to be for maintainers of the module. That means receiving a number of emails, communications, issues and pull-requests (like this very one), etc meant for module maintainers, becoming up to speed with maintainers' docs, rules, standards, etc. Joe Benden is phasing out of that role, and I am stepping in - I still gotta read a **lot** of maintainers' documentation myself so I am unsure of way too many things yet.

If that is so and you're OK with it, I would be more than happy to have you along :-).

Reply:
Hmm, not sure I'm up for that. I noticed this change in the latest change and figured it was a "contributors" list. Let me mull it over (whether I'll have the bandwidth to take on more responsibility) and maybe remove this in the next few days if I decide I don't.